### PR TITLE
nginx_conf.sh is changed-final

### PR DESCRIPTION
**Pull request by Frida199**
Please check!!!

### CODE REVIEW

File: nginx_conf.sh [hunk 1]
```diff
@@ -1,28 +1,34 @@
 #!/bin/bash
 
-# Updating and installing required commands
-apt update  -y
+# installing required commands
 apt install wget -y
 apt install curl -y
```
Comment (mkrtchyan-t) [inline nginx_conf.sh:13]: before installing check if *wget* and *curl* are already installed

File: nginx_conf.sh [hunk 2]
```diff
@@ -1,28 +1,34 @@
...
+response=$(curl --write-out %{http_code} --silent --output /dev/null $url)
+file="bootcamp_aca.conf"
 # Checking if the given URL works and if we already have one
 if [[ $response = 200 ]]; then
-	wget $URL
+	wget $url
+fi
+
+# chechking if file is zero or not 
+if [[ -z $file ]]; then
```
Comment (mkrtchyan-t) [inline nginx_conf.sh]: change
```
if [[ ! -z $file ]]; then 
```

File: nginx_conf.sh [hunk 3]
```diff
@@ -1,28 +1,34 @@
...
+if [[ -f /etc/nginx/sites-enable/default ]]
+then
+rm /etc/nginx/sites-enabled/default
 
-if [[ $deff -ne 0 ]]; then 
-       cp bootcamp_aca.conf /etc/nginx/sites-enabled/default
 fi
 
+#testing nginx
+nginx -t
```
Comment (mkrtchyan-t) [inline nginx_conf.sh:39]: you should check if the error code is 1 or 0 (`echo $?`)
if 0 -> then everything is OK
if 1 -> nginx testing failed

**Review by mkrtchyan-t — CHANGES_REQUESTED** [review]

File: nginx_conf.sh (hunk 1 above)
Comment (Frida199) [reply nginx_conf.sh:13]: done

File: nginx_conf.sh (hunk 2 above)
Comment (Frida199) [reply nginx_conf.sh]: done

File: nginx_conf.sh (hunk 3 above)
Comment (Frida199) [reply nginx_conf.sh:39]: done

Comment (mkrtchyan-t) [reply nginx_conf.sh:39]: do `else exit`

**Review by mkrtchyan-t — APPROVED** [review]
approved !

**Review by sargsyan1986 — APPROVED** [review]
I hope its OK!

**Pull request merged**